fhir-persistence-schema/README.md

```diff
@@ -11,8 +11,12 @@ The executable command line interface (cli) version of this module can be downlo
 
 To create the Db2 database and database user, use the following commands:
 
+> **Caveat 1:** When creating the user, make sure there is no group with the same name; otherwise the step *Grant privileges to data access user* below will fail with `SQLCODE=-569, SQLSTATE=56092` ("Authorization ID does not uniquely identify a user, a group or a role in the system"). If there already exists a group with the same name as the user, consider renaming the group with `groupmod -n <new-name> <old-name>`.
+>
+> **Caveat 2:** When creating the database, `PAGESIZE` is important. So *do* use the statement below and not, e.g., the environment variable `DBNAME` of the [Db2 Docker container](https://hub.docker.com/r/ibmcom/db2) to generate the database; otherwise the step *Deploy new schema* below will fail with `SQLCODE=-286, SQLSTATE=42727`.  
+
```

Comment:
Feels more like errata to me, and I'd opt not to use inline caveats.  I'd suggest pushing to the bottom and providing a reference later.

Reply:
Remember, no one really reads the docs ;-) Even if the hints are placed only one line after the command to execute, in my experience, they will not be read by everyone.

These are real-life pitfalls that non-IBM users will fall in.

Reply:
I'd prefer not to have the caveats inline, and use the footnote.